.gitattributes

```diff
@@ -1,3 +1,40 @@
-* text=auto eol=lf
+# Set default behavior to automatically normalize line endings.
```

Comment:
```suggestion
# Ensure consistent line endings between *nix and Windows to avoid unexpected issues
# due to CRLF/LF differences between those two operating system camps.
# Set default behavior to normalize line endings automatically.
```